util/admin/admin.go

```diff
@@ -87,9 +87,14 @@ func GetDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
 func IsJobRollbackable(job *model.Job) bool {
 	switch job.Type {
 	case model.ActionDropIndex, model.ActionDropPrimaryKey:
-		// We can't cancel if index current state is in StateDeleteOnly or StateDeleteReorganization, otherwise will cause inconsistent between record and index.
+		// We can't cancel if index current state is in StateDeleteOnly or StateDeleteReorganization or StateWriteOnly, otherwise will cause inconsistent between record and index.
```

Comment:
```suggestion
		// We can't cancel if index current state is in StateDeleteOnly or StateDeleteReorganization or StateWriteOnly, otherwise these will be an inconsistent issue between record and index.
```

Reply:
these or there ?

Reply:
there